cartographer_ros/CMakeLists.txt

```diff
@@ -137,17 +149,21 @@ install(DIRECTORY configuration_files/
 )
 
 install(TARGETS
-  cartographer_rviz_submaps_visualization
+  cartographer_rviz_submaps_visualization cartographer_node
   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
 
 install(FILES
-  plugin_description.xml
+  rviz_plugin_description.xml
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
 
 install(DIRECTORY ogre_media/
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/ogre_media}
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/ogre_media
 )
+
+install(PROGRAMS
+         scripts/imu_pub.py
```

Comment:
This should no longer be needed. If you disable the IMU in the configuration, it's no longer subscribed to.